requirements.txt

```diff
@@ -3,12 +3,11 @@ click
 git+https://github.com/matrix-org/matrix-python-sdk.git
 # use czepluch's branch that implements MS contract
 # git+https://github.com/raiden-network/raiden-contracts.git
-git+https://github.com/czepluch/raiden-contracts.git@ms_smart_contract
+git+https://github.com/raiden-network/raiden-contracts.git
 git+https://github.com/raiden-network/raiden-libs.git
 gevent
 jsonschema
-eth-utils
-coincurve
-ethereum
-eth-tester
-web3>=4.0.0b1
+eth-utils>=1.0.1,<2.0
+coincurve>=7.1.0,<8.0
+ethereum>=2.3.1,<3.0
```

Comment:
Just wondering: is this still needed?

Reply:
not sure, these dependencies were copypasted from `raiden-libs`